deploy/complete/docker-compose/docker-compose.yml

```diff
@@ -105,12 +109,12 @@ services:
     environment:
       - OADB_USER=carts_user
       - OADB_PW=Default_Password123#
-      - OADB_SERVICE=mcarts_tp
+      - OADB_SERVICE=mock
```

Comment:
Carts in this configuration is live. the container will run and accept requests. Nothing is persisted though, since this the persistence is mocked (in memory).